src/bin/cql-stress-cassandra-stress/settings/option/node.rs

```diff
@@ -103,6 +109,12 @@ fn prepare_parser() -> (ParamsParser, NodeParamHandles) {
         "Preferred datacenter for the default load balancing policy",
         false,
     );
+    let shard_connection_count = parser.simple_param(
+        "connections=",
```

Comment:
Does the original c-s have an option like that? If so, I'd suspect that it specifies per-node connection count, not per-shard. I'd rather use a different name that suggests that it's a per-shard count.

Reply:
No, there is no such option in the original c-s. I agree with the name change.